dialoptions.go

```diff
@@ -401,6 +403,14 @@ func WithStatsHandler(h stats.Handler) DialOption {
 	})
 }
 
+// WithBinaryLogger returns a DialOption that specifies the binary logger for
+// this ClientConn.
+func WithBinaryLogger(bl binarylog.Logger) DialOption {
```

Comment:
Please make this internal-only:

```go
package internal

// WithBinaryLogger yadd yadda
var WithBinaryLogger interface{}  // func(binarylog.Logger) grpc.DialOption
```
```go
package grpc

func withBinaryLogger(....) .....

func init() { internal.WithBinaryLogger = withBinaryLogger }
```

(And the server option too)

Reply:
Switched. (Won't allow me to respond in review lol)